frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -269,6 +270,48 @@
 
         utilities.sendRequest(parameters);
 
+        vm.toggleParticipation = function (ev, isRegistrationOpen) {
+            // ev.stopPropagation();
+            var participationState;
+            if (isRegistrationOpen) {
+                participationState = 'close';
+            } else {
+                participationState = 'open';
+            }
+            var confirm = $mdDialog.confirm()
+                          .title('You really want to ' + participationState + ' the participation?')
```

Comment:
`'You really want to ' + participationState + ' the participation?'`
 -> `'participationState + ' participation in the challenge?'`

Reply:
done